src/Data/Unrestricted/Internal/Instances.hs

```diff
@@ -142,6 +146,18 @@ instance (Dupable a, Dupable b, Dupable c) => Dupable (a, b, c) where
 instance (Movable a, Movable b, Movable c) => Movable (a, b, c) where
   move (a, b, c) = (,,) Data.<$> move a Data.<*> move b Data.<*> move c
 
+-- XXX: The Consumable and Dupable instances for V will be easier to define (in
+-- fact direct, we may consider adding a deriving-via combinator) when we have a
+-- traversable-by-a-data-applicative class see #190.
```

Comment:
This is a confusing comment but I'm going to assume there's a clear explanation of what it means in #190.